libraries/compile-examples/entrypoint.sh

```diff
@@ -39,7 +39,15 @@ ln -s $PWD $HOME/Arduino/libraries/.
 
 # Find all the examples and loop build each
 EXAMPLES=`find examples/ -name '*.ino' | xargs dirname | uniq`
+# Set default exit status
+SCRIPT_EXIT_STATUS=0
 for EXAMPLE in $EXAMPLES; do
   echo Building example $EXAMPLE
   arduino-cli compile --verbose --warnings all --fqbn $FQBN $EXAMPLE
-done || exit 1
\ No newline at end of file
+  ARDUINO_CLI_EXIT_STATUS=$?
+  if [[ $ARDUINO_CLI_EXIT_STATUS -ne 0 ]]; then
+    SCRIPT_EXIT_STATUS=$ARDUINO_CLI_EXIT_STATUS
```

Comment:
That's correct. I thought it would be best not to fail the run on the first compilation error because then it can identify any problems with compiling the other examples as well.

Reply:
Good thinking :+1: as I see no other issues I shall merge this PR.